pkg/database/bigquery.schema.json

```diff
@@ -82,5 +82,22 @@
     "mode": "REPEATED",
     "name": "nodes",
     "type": "RECORD"
+  },
+  {
+    "fields": [
+      {
+        "mode": "REQUIRED",
```

Comment:
Good point. IMO both this field and the `value` field should be required, if for no other reason than to maintain some consistency and sanity in our data. Technically, I think that empty string `""` should be a valid value for both the extension `name` and `value` fields, since after all `{"":""}` is valid JSON. `""` is the zero value for strings, so I don't think that any extra enforcement should be required to ensure that extensions have the correct structure.